analyses/focal-cn-file-preparation/01-prepare-cn-file.R

```diff
@@ -296,6 +371,36 @@ if (xy_flag) {
                       by = c("biospecimen_id" = "Kids_First_Biospecimen_ID")) %>%
     dplyr::select(-germline_sex_estimate, dplyr::everything())
 
+  #### Wrangle GISTIC data ---------------------------------------------
+  
+  # Transpose the GISTIC data.frame
+  sex_chrom_transposed_gistic_df <- gistic_df %>%
```

Comment:
I think overall you can combine this section with the one above and then just split the autosomes and sex chromosomes in their own file at the end. I'd take a look into that so we can shorten this up and maybe speed it up slightly. Though I know this isn't the time consuming part.